docs/windows.md

```diff
@@ -439,7 +449,10 @@ steps. And then run the following commands in powershell.
 mkdir c:\k\antrea
 cd c:\k\antrea
 curl.exe -LO https://github.com/antrea-io/antrea/releases/download/<TAG>/Start-AntreaAgent.ps1
+# Run antrea-agent without kube-proxy
 # $KubeConfigPath is the path of kubeconfig file
+./Start-AntreaAgent.ps1 -kubeconfig $KubeConfigPath -StartKubeProxy $false
+# Run Antrea-Agent with kube-proxy (deprecated since Kubernetes 1.26)
 ./Start-AntreaAgent.ps1 -kubeconfig $KubeConfigPath
```

Comment:
After your patch, the default value of `$StartKubeProxy` is "$false", so line 456 actually does not start kube-proxy.

Reply:
Updated.